src/OrchardCore.Modules/OrchardCore.Alias/Handlers/AliasPartHandler.cs

```diff
@@ -53,20 +49,15 @@ public async override Task UpdatedAsync(UpdateContentContext context, AliasPart
                 templateContext.SetValue("ContentItem", part.ContentItem);
 
                 part.Alias = await _liquidTemplateManager.RenderAsync(pattern, NullEncoder.Default, templateContext);
-                part.Apply();
-            }
-        }
+                part.Alias = part.Alias.Replace("\r", String.Empty).Replace("\n", String.Empty);
```

Comment:
```csharp
part.Alias = part.Alias.Replace ("\r\n", String.Empty);
```

Reply:
Nope. won't work on linux if we do that.

Reply:
What about

```csharp
part.Alias = part.Alias.Replace (Environment.NewLine, String.Empty);
```

Reply:
Somehow I feel there may still be a problem introduced by using `Environment.NewLine` and adds little advantage

Also it is consistent with elsewhere in the code base (see the `AutorouteHandler`)